systers_portal/community/models.py

```diff
@@ -37,6 +37,7 @@ class Community(models.Model):
     class Meta:
         verbose_name_plural = "Communities"
         permissions = (
+            ('add_tag', 'Add blog tags'),
```

Comment:
Those permissions are row level (tied to a Community object). We don't need a row level permission to add tags. Tags are shared among all communities. Additionally the "add_tag" permission already exists and was generated automatically by Django. You can see it if you login to the admin panel.